test/fixtures/es-modules/unexpected-export-1.cjs

```diff
@@ -0,0 +1,2 @@
+const example = 10;
+export default example;
```

Comment:
Please make sure to include separate tests for the cases the regular expression is matching.